ignite/contrib/handlers/lr_finder.py

```diff
@@ -263,11 +276,15 @@ def plot(
             lrs = lrs[skip_start:-skip_end]
             losses = losses[skip_start:-skip_end]
 
+        # Check to display legends
+        if display_legends:
+            plt.legend(legends)
         # Plot loss as a function of the learning rate
         plt.plot(lrs, losses)
         if log_lr:
             plt.xscale("log")
-        plt.xlim([lrs[0], lrs[-1]])
+        if not isinstance(lrs[0], list):
+            plt.xlim([lrs[0], lrs[-1]])
```

Comment:
Yes we can, but It may look something like this
![dummy_muliple_param_groups](https://user-images.githubusercontent.com/54319724/116797669-968eaf00-aae8-11eb-80e8-6dfdd43bb9ad.jpg)
That's why I thought we don't need it, What do you think?

Reply:
This looks a bit strange as blue, orange curves are also cut even if we cut only LR...

Reply:
I think it's not strange, as we cut from the minimum LR to the max one

Reply:
OK, I think I understand now why it is so. I think it is OK to have the plot like that as the docs on skip_start and skip_end explicitly says that we skip batches. 

> Yes we can, but It may look something like this

OK, let's display it like that.